src/libOpenImageIO/exif.cpp

```diff
@@ -280,8 +280,14 @@ class TagMap {
         for (int i = 0;  tag_table[i].tifftag >= 0;  ++i) {
             const EXIF_tag_info *eti = &tag_table[i];
             m_tagmap[eti->tifftag] = eti;
-            if (eti->name)
-                m_namemap[eti->name] = eti;
```

Comment:
Finally^2 -- these table seem small enough that I wonder if `std::map` is needed at all. I bet a good old linear search would run just as fast or faster.